src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -86,6 +86,9 @@
     HOME_OFFICE_BUNDLE(
         "homeOfficeBundle", new TypeReference<List<IdValue<DocumentWithDescription>>>(){}),
 
+    UPLOADED_HOME_OFFICE_BUNDLE_DOCS(
+        "uploadedHomeOfficeBundleDocs", new TypeReference<List<IdValue<String>>>(){}),
```

Comment:
I think, it should be `String` not `List<>`. Above will work unless you read the value in code

Reply:
Thanks Marcin. It's fixed now